index.ts

```diff
@@ -4,11 +4,13 @@ config();
 import * as fs from "fs";
 import * as z from "zod";
 import clerkClient from "@clerk/clerk-sdk-node";
-import ora, { Ora } from "ora";
+import Listr from "listr";
 
 const SECRET_KEY = process.env.CLERK_SECRET_KEY;
-const DELAY = Number(process.env.DELAY ?? 1_000);
+const DELAY = Number(process.env.DELAY ?? 2_000);
+const RATE_LIMIT_DELAY = Number(process.env.RATE_LIMIT_DELAY ?? 5_000);
 const IMPORT_TO_DEV = process.env.IMPORT_TO_DEV_INSTANCE ?? "false";
+const BATCH = Number(process.env.BATCH ?? 10);
```

Comment:
How many users will be pulled off the queue and migrated concurrently. Larger batches means longer cooldowns. Theoretically the script can be tuned for performance by optimizing the batch and cooldown and perfectly hit the rate limit.